database/db_connection.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+require('env2')('./congif.env');
+
+const db = mongoose.connection;
+mongoose.connect(process.env.MONGODB_URI, {
```

Comment:
do we need a whole different file for this ??
why not justg add it to the src/index.js ( which should be renamed )